src/main/java/com/minecolonies/coremod/client/gui/WindowBuildingBrowser.java

```diff
@@ -212,6 +220,8 @@ private void rebuildCache()
             return;
         }
 
+        final List<Block> browsableBlocks = findBrowsableBlocks();

```

Comment:
is this cached?

Reply:
Not inherently, but it's only called once per login since the method as a whole is part of the cache rebuild.

Reply:
kk